terraform_files/vsphere/variables-vsphere.tf

```diff
@@ -0,0 +1,91 @@
+//////
+// vSphere variables
+//////
+
+variable "vsphere_vcenter" {
+  type        = string
+  description = "vSphere vcenter server ip address or fqdn (vCenter server name for vSphere API operations)"
+}
+
+variable "vsphere_username" {
+  type        = string
+  description = "vSphere vcenter server username"
+}
+
+variable "vsphere_password" {
+  type        = string
+  description = "vSphere vcenter server username"
+}
+
+variable "cluster_name" {
+  type = string
+  description = <<EOF
+AI cluster name
+All the resource will be located under a folder with this name
+and tagged with the cluster name
+The resources should be associate with this name for easy recognition.
+EOF
+}
+
+variable "masters_count" {
+  type = string
+  default = "3"
+  description = "The number of master nodes to be created."
+}
+
+variable "iso_download_path" {
+  type        = string
+  description = "The ISO path (We have to upload this file to the vsphere)"
+  default = "dummy"
+}
+
+variable "vsphere_cluster" {
+  type        = string
+  description = "vSphere cluster name"
```

Comment:
Explain what's the different usage between the AI cluster name and vSphere cluster name

Reply:
vsphere cluster is a cluster of hosts not applicative cluster.

Reply:
Don't explain it to me.. Write it on the description

Reply:
I think we shouldn't describe vsphere object types but i did